### DISCUSSION TITLE
Error message contains unexpected prefix: "input: path "

### DISCUSSION THREAD

**Discussion by rot1024** [99designs/gqlgen#1797] [Q&A]
### What happened?

Errors of a query are displayed as following even when using custom ErrorPresenter:

```json
{
  "errors": [
    {
      "message": "input: signup existed user",
      "path": [
        "signup"
      ]
    }
  ],
  "data": {
    "signup": null
  }
}
```

We can see `input: signup ` prefix.

The error returned in resolvers is a simple error: `errors.New("existed users")`

### What did you expect?

I want to display ONLY the message part.

```json
{
  "errors": [
    {
      "message": "existed user",
      "path": [
        "signup"
      ]
    }
  ],
  "data": {
    "signup": null
  }
}
```

This behavior is occurred by https://github.com/vektah/gqlparser/blob/master/gqlerror/error.go#L40 And we cannot change this error format even when using ErrorPresenter. Is that expected? 

### versions
 - `gqlgen version`? 0.13.0
 - `go version`? 1.16.2
 - dep or go modules? go modules

**Comment by iFurySt** [comment]
I do it in this strange way:
```go
var re = regexp.MustCompile(`input: \w+ (.*)`)
match := re.FindStringSubmatch(e.Error())
fmt.Println(match[1])
```
😳
